src/test/java/org/cactoos/experimental/TimedTest.java

```diff
@@ -84,7 +84,7 @@
         "PMD.AvoidThrowingRawExceptionTypes", "PMD.DoNotThrowExceptionInFinally"
     })
     void containsResults() throws Exception {
-        new RepeatedCallable<>(
+        new Repeated<>(
```

Comment:
@victornoel Can this type of test be migrated to `@RepeatedTest` annotation?

Reply:
@andreoss I've added a todo, thx for the idea